code/modules/admin/verbs/striketeam.dm

```diff
@@ -107,98 +119,74 @@ GLOBAL_VAR_INIT(sent_strike_team, 0)
+		P.name = "Special Operations Manual"
+		P.icon_state = "paper"
 		var/obj/item/stamp/centcom/stamp = new
 		P.stamp(stamp)
 		qdel(stamp)
 
-	for(var/thing in GLOB.landmarks_list)
+	for(var/thing in GLOB.landmarks_list) // I dont think this does anything anymore, should remove it?
 		var/obj/effect/landmark/L = thing
 		if(L.name == "Commando-Bomb")
 			new /obj/effect/spawner/newbomb/timer/syndicate(L.loc)
 			qdel(L)
 
-	message_admins("<span class='notice'>[key_name_admin(usr)] has spawned a CentComm DeathSquad.</span>", 1)
+	message_admins("<span class='notice'>[key_name_admin(usr)] has spawned a CentComm DeathSquad.</span>")
 	log_admin("[key_name(usr)] used Spawn Death Squad.")
 	return 1
 
 /client/proc/create_death_commando(obj/spawn_location, is_leader = FALSE)
 	var/mob/living/carbon/human/new_commando = new(spawn_location.loc)
 	var/commando_leader_rank = pick("Lieutenant", "Captain", "Major")
 	var/commando_name = pick(GLOB.commando_names)
+	var/obj/item/organ/external/head/head_organ = new_commando.get_organ("head")
```

Comment:
Add a comment that this appearance thing is basically copy-pasted from ERT code.